internal/file/file_service_operator.go

```diff
@@ -277,7 +277,7 @@ func (fso *FileServiceOperator) UpdateFile(
 	return fso.sendUpdateFileStream(ctx, fileToUpdate, fso.agentConfig.Client.Grpc.FileChunkSize)
 }
 
-func (fso *FileServiceOperator) MoveFilesFromTempDirectory(
+func (fso *FileServiceOperator) MoveFileFromTempDirectory(
 	ctx context.Context, fileAction *model.FileCache, tempDir string,
 ) error {
```

Comment:
```suggestion
func (fso *FileServiceOperator) RenameFile(
ctx context.Context, sourceFile, destination string,
) error {
```

I think calling the function rename instead of move makes it more clear in what its doing